docs/platforms/javascript/common/user-feedback/configuration/index.mdx

```diff
@@ -18,12 +18,13 @@ The following configuration options apply to version 8.0.0 and above of the Java
-| `id`           | `string`                                   | `sentry-feedback` | The `id` attribute of the `<div>` that contains the feedback widget. See [CSS Customization](#css-customization) for more.                                                                                                                 |
+| Key            | Type                                                                                      | Default           | Description                                                                                                                                                                                                                                |
+| -------------- | ----------------------------------------------------------------------------------------- | ----------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ |
+| `autoInject`   | `boolean`                                                                                 | `true`            | Injects the Feedback widget into the application when the integration is added. Set `autoInject: false` if you want to call `feedback.attachTo()` or `feedback.createWidget()` directly, or only want to show the widget on certain views. |
+| `showBranding` | `boolean`                                                                                 | `true`            | Displays the Sentry logo inside of the form.                                                                                                                                                                                               |
+| `colorScheme`  | <code>"system" \| "light" \| "dark"</code>                                                | `"system"`        | Shows the color theme choices. `"system"` will use your OS color scheme.                                                                                                                                                                   |
+| `id`           | `string`                                                                                  | `sentry-feedback` | The `id` attribute of the `<div>` that contains the feedback widget. See [CSS Customization](#css-customization) for more.                                                                                                                 |
+| `tags`         | `Record<string, number \| string \| boolean \| bigint \| symbol \| null \| undefined>` | `{}`              | [Tags](/platforms/javascript/enriching-events/tags/) to set on the feedback event.                                                                                                                                                                                                         |
```

Comment:
@ryan953 you could use the PlatformLink component, instead of a markdown link. It would then link to the page for the base platform, but also on its guides. Otherwise, when you view the page for e.g. React, the link would be hardcoded to link to the page for the base platform.